NURESTConnection.j

```diff
@@ -149,18 +149,24 @@ var NURESTConnectionLastActionTimer,
     var responseObject   = [[aConnection responseData] JSONObject],
         responseCode     = [aConnection responseCode],
         containsInfo     = (responseObject && responseObject.errors),
-        errorName,
-        errorDescription;
+        errorMessages         = @[];
+
+    var errorName, errorDescription;
```

Comment:
`errorName` and `errorDescription` are used later in the `case` blocks, but in fact they are redefined. I'll remove them from here.